KifuSF/InformationModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  InformationModel.swift
+//  KifuSF
+//
+//  Created by Alexandru Turcanu on 04/01/2019.
+//  Copyright © 2019 Alexandru Turcanu. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - InformationModel
+struct InformationModel {
+    // MARK: - Variables
+    private let information: String
+
+    var cellTitle: String
+    var errorAlertController: UIAlertController
+    
+    // MARK: - Initializers
+    init(cellTitle: String, information: String, errorMessage: String? = nil) {
+        self.cellTitle = cellTitle
+        self.information = information
+        self.errorAlertController = UIAlertController(errorMessage: errorMessage)
+    }
+}
+
+// MARK: - SettingsItemProtocol
+extension InformationModel: SettingsItem {
+    var viewControllerToShow: UIViewController {
+        let viewController = UIViewController()
+        let textView = UITextView(
+            font: UIFont.preferredFont(forTextStyle: .body),
+            textColor: UIColor.Text.SubHeadline
+        )
+
+        viewController.view.addSubview(textView)
+        viewController.view.directionalLayoutMargins = NSDirectionalEdgeInsetsMake(16, 16, 16, 16)
+
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.autoPinEdgesToSuperviewMargins()
+
+        textView.text = information
+        textView.alwaysBounceVertical = true
+
+        viewController.title = cellTitle
+        viewController.view.backgroundColor = UIColor.Pallete.White
+
+        return viewController
+    }
+
+    func didSelectItem(in viewController: UIViewController) {
+        viewController.navigationController?.pushViewController(viewControllerToShow, animated: true)
+    }
```

Comment:
this presenting should be handled by the ViewController (`SettingsViewController`) since it MAY not always be presented by a navigation controller

Reply:
Look [here](https://github.com/Kifuapp/KifuSF/pull/25/files#r245427402) for suggested changes

Reply:
I get your point, but in some cases, you need to push a new ViewController using the navigation controller other times you just modally present it (ex: SFSafariViewController), so it should be left up to the class/struct that is conforming to the protocol to have their own implementation.